projects/jobe/Dockerfile

```diff
@@ -12,59 +12,58 @@ LABEL \
     org.opencontainers.image.documentation="https://github.com/trampgeek/jobeinabox" \
     org.opencontainers.image.source="https://github.com/trampgeek/jobeinabox"
 
-ARG TZ=Etc/GMT-0
```

Comment:
I think we want to be in GMT so jobe is in the same timezone as the rest of the servers